src/cool_seq_tool/sources/uta_database.py

```diff
@@ -271,68 +270,30 @@ def _transform_list(li: list) -> list[list[Any]]:
         """
         return [list(i) for i in li]
 
-    async def get_genes_and_alt_acs(
+    async def get_genomic_ac_genes(
         self,
         pos: int,
-        strand: Strand | None = None,
-        chromosome: int | None = None,
-        alt_ac: str | None = None,
-        gene: str | None = None,
-    ) -> tuple[GenesGenomicAcs | None, str | None]:
-        """Return genes and genomic accessions for a position on a chromosome or alt_ac
+        alt_ac: str,
+    ) -> tuple[set[str] | None, str | None]:
+        """Get genes given a genomic accession and position
 
         :param pos: Genomic position
-        :param strand: Strand
-        :param chromosome: Chromosome. Must give chromosome without a prefix
-            (i.e. ``1`` or ``X``). If not provided, must provide ``alt_ac``.
-            If ``alt_ac`` is also provided, ``alt_ac`` will be used.
-        :param alt_ac: Genomic accession (i.e. ``NC_000001.11``). If not provided,
-            must provide ``chromosome``. If ``chromosome`` is also provided, ``alt_ac``
-            will be used.
-        :param gene: Gene symbol
-        :return: Genes and genomic accessions and warnings if found
+        :param alt_ac: RefSeq
```

Comment:
```suggestion
        :param alt_ac: RefSeq accession, e.g. ``"NC_000007.14"``
```

Reply:
I was going to suggest some code block examples here but the async thing is annoyingly verbose